### DISCUSSION TITLE
Export to HTML but With Slides One Below the Other?

### DISCUSSION THREAD

**Discussion by stungeye** [marp-team/marp#445] [Q&A] [answered]
Is it possible with marp-cli to export slides to HTML, but instead of the output showing just one slide at a time, the slides are display one below the other? In other words, is it possible to export to one long HTML page? (The look of this long HTML page would be similar to what one sees in the VS Code preview when using the Marp for VS Code extension.)

**Comment by yhatt** [comment, marked as answer]
[Try using the `bare` template in Marp CLI.](https://github.com/marp-team/marp-cli#bare-template) It has no extra features, and just lines up rendered slide elements in HTML. You can still navigate slide pages by scrolling up and down.

```
marp --template bare your-slide.md
```

**Reply by stungeye** [reply]
Perfect! Thanks for the quick response. And thank you for this amazing tool! It's really incredible. I appreciate all the work you and other Marp contributors have put into this project.
